src/TConsole.cpp

```diff
@@ -680,16 +680,14 @@ void TConsole::refresh()
     QApplication::sendEvent(this, &event);
 }
 
-void TConsole::clearSelection(bool yes) const
+void TConsole::clearSelection() const
 {
-    if (yes) {
```

Comment:
You should remove the indentation - even though it makes a bigger change - so that the indentation is correct after the change.